lib/actions/scrollTo.js

```diff
@@ -4,10 +4,10 @@ const { defaultConfig, setNavigationOptions } = require('../config');
 const { highlightElement } = require('../elements/elementHelper');
 const { doActionAwaitingNavigation } = require('../doActionAwaitingNavigation');
 
-const scrollToElement = async (element) => {
+const scrollToElement = async (element, alignments) => {
   function scrollToNode() {
     const element = this.nodeType === Node.TEXT_NODE ? this.parentElement : this;
-    element.scrollIntoView({ block: 'nearest', inline: 'nearest' });
+    element.scrollIntoView({ block: alignments.block, inline: alignments.inline });
```

Comment:
Does it make sense to make this as follows?

```suggestion
    element.scrollIntoView(alignment);
```

As mentioned here https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView

alignment has other options too